creme/activities/forms/fields.py

```diff
@@ -60,6 +60,14 @@ def limit_choices_to(self, limit_choices_to):
         """
         self.enum.enumerator.limit_choices_to = limit_choices_to
 
+    def __deepcopy__(self, memo):
+        """
+        New deepcopy for the 'limit_choices_to' property.
+        """
```

Comment:
I do not find this doc-string very relevant (& why "New" ?)...

Reply:
"Add" is a better word I agree